<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingMain</class>
 <widget class="QWidget" name="TestingMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(27, 150, 208);
}
QPushButton{ 
border-width:0;
border-style:outset;
background: transparent;
font-size:26px;
color:white;
}
QToolButton{
background-color:transparent;
border:noborder;
font-size:26px;
color:white;
}</string>
  </property>
  <widget class="QLabel" name="testing_lb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190,210, 240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; color:#ffff00;&quot;&gt;测 试&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>671</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolButton" name="toolButton_airvalve">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>1.气阀测试</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/VALVE.BMP</normaloff>:/img/VALVE.BMP</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="toolButton_actuator">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>2.选针器测试</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/NEEDLE.BMP</normaloff>:/img/NEEDLE.BMP</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButton_stepmotor">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>3.步进电机测试</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/MOTOR.BMP</normaloff>:/img/MOTOR.BMP</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="toolButton_input">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>4.输入测试</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/INPUT.BMP</normaloff>:/img/INPUT.BMP</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="toolButton_output">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>5.输出测试</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/OUTPUT.BMP</normaloff>:/img/OUTPUT.BMP</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButton_back">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>6.返回</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/BACK.BMP</normaloff>:/img/BACK.BMP</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Resources/picture.qrc"/>
 </resources>
 <connections/>
</ui>
